### ISSUE TITLE
Add NewCrop Menu items when enabled to JSON menu file

### ISSUE THREAD

**Issue by tmccormi** [LibreHealthIO/LibreEHR#698]
The NewCrop admin menu items seem to have been missed (thx for the heads up @teryhill)

**Comment by aethelwulffe** [comment]
I am working on the JSON file right now...can look.

**Comment by tmccormi** [comment]
Ok, thx

**Comment by aethelwulffe** [comment]
Hunting down the links from a copy of left-nav, then figuring out the 
ACL bits...

**Comment by aethelwulffe** [comment]
```
  <?php
  if($GLOBALS['erx_enable']) {
    $newcrop_user_role = sqlQuery("SELECT newcrop_user_role FROM users WHERE username = '".$_SESSION['authUser']."'");
    if($newcrop_user_role['newcrop_user_role']) {
  ?>
  <li><a class="collapsed" id="feeimg" ><span><?php xl('New Crop','e') ?></span></a>
    <ul>
      <li><a class="collapsed_lv2"><span><?php xl('Status','e') ?></span></a>
        <ul>
          <?php genTreeLink('RTop','erx',xl('e-Rx')); ?>
          <?php genMiscLink('RTop','err','0',xl('e-Rx Renewal'),'eRx.php?page=status'); ?>
          <?php if($newcrop_user_role['newcrop_user_role'] === 'erxadmin') genMiscLink('RTop','ere','0',xl('e-Rx EPCS'),'eRx.php?page=epcs-admin'); ?>
        </ul>
      </li>
    </ul>
  </li>
  <?php
    }
  }
  ?>
```
And
```
          <?php
          if($newcrop_user_role['newcrop_user_role'] || $GLOBALS['erx_enable']) {
            if (acl_check('admin', 'users'   )) genMiscLink('RTop','adm','0',xl('eRx Logs'),'logview/erx_logview.php');
          }
          ?>
```
from the array elements of
```
  'erx' => array(xl('e-Rx') , 1, 'eRx.php'),
  'err' => array(xl('e-Rx Renewal') , 1, 'eRx.php?page=status'),
  'ere' => array(xl('e-Rx EPCS') , 1, 'eRx.php?page=epcs-admin'),
```
Should be all of it...
I will see how it does as a top-level (optional) menu item.  Might as well try the menu editor while I have it open...

**Comment by teryhill** [comment]
Would this help?

  {
    "label": "New Crop",
    "menu_id": "feeimg",
    "children": [
      {
        "label": "e-Rx",
        "menu_id": "ncr0",
        "target": "pat",
        "url": "/interface/eRx.php",
        "children": [],
        "requirement": 1,
        "acl_req": ["patients","rx"]
      },
      {
        "label": "e-Rx Renewal",
        "menu_id": "ncr1",
        "target": "pat",
        "url": "/interface/eRx.php?page=status",
        "children": [],
        "requirement": 1,
        "acl_req": ["patients","rx"]
      },
      {
        "label": "e-Rx EPCS",
        "menu_id": "ncr2",
        "target": "pat",
        "url": "/interface/eRx.php?page=epcs-admin",
        "children": [],
        "requirement": 0,
        "acl_req": ["patients","rx"],
        "global_req": "newcrop_user_role_erxadmin"
      }
    ],
    "requirement": 1,
    "global_req_strict": ["erx_enable","newcrop_user_role"]
  },

**Comment by aethelwulffe** [comment]
If you are done...do it!  Woohoo!  Things are getting done before I can touch them!

**Comment by tmccormi** [comment]
zhh prob used  feimg because they have no imagination

**Comment by tmccormi** [comment]
I bet we can find and Rx image

**Comment by teryhill** [comment]
I got the stuff in place but I cannot get it to show the menu . I will send the pr and let you guys look at it. The feeimg is there because of where it originated 8^)

**Comment by teryhill** [comment]
PR there

**Comment by teryhill** [comment]
I think it is something in the menu program that is keeping it from appearing. I can see everything in the editor and it looks ok. may be @azwreith  or @nileshprasad137  can offer some hint.

**Comment by aethelwulffe** [comment]
I had just added a new top level item to the menu without using the fancy editor thingie...I should be able to figure out why this is stalled on you.

**Comment by aethelwulffe** [comment]
Got the ACL fixed on #699, and is ready for final review and merge.
